app/controllers/users.js

```diff
@@ -82,3 +85,133 @@ exports.signin = (request, response, next) => {
       next(err);
     });
 };
+
+exports.listUsers = (request, response, next) => {
+  const userLogged = request.user;
+  if (userLogged) {
+    const limit = request.query.limit ? parseInt(request.query.limit) : 10;
+    const offset = request.query.offset ? parseInt(request.query.offset) : 0;
+
+    userService.getAllUsersPaginate(limit, offset).then(users => {
+      if (users != null) {
+        userService.countAllUsers().then(count => {
+          const usersFiltered = users.map(user => {
+            return {
+              id: user.id,
```

Comment:
Can't you return the entire object? i don't know if there's some extra field you want to omit, if that's not the case you can just return `user` here

Reply:
is to avoid showing the password even if it is encrypted